### PR TITLE
Installation instructions

### PR DESCRIPTION
**Pull request by kristinawlai**
Review updates to installation instructions

### CODE REVIEW

**Review by d-morrison — CHANGES_REQUESTED** [review]
This branch shouldn't include the new vignette ('typhoid-seroincidence.Rmd'); let's remove it and see if that resolves the check errors?

**Comment by kristinawlai** [comment]
> This branch shouldn't include the new vignette ('typhoid-seroincidence.Rmd'); let's remove it and see if that resolves the check errors?

I reverted the new vignette changes (i.e. removed it from this branch). It seems like there are still problems with 'cover.Rmd' and 'tutorial.Rmd'?

**Comment by d-morrison** [comment]
> > This branch shouldn't include the new vignette ('typhoid-seroincidence.Rmd'); let's remove it and see if that resolves the check errors?
> 
> I reverted the new vignette changes (i.e. removed it from this branch). It seems like there are still problems with 'cover.Rmd' and 'tutorial.Rmd'?

I'm working on this today; not sure yet what's causing these errors.

File: vignettes/installation.Rmd [hunk 1]
```diff
@@ -83,19 +83,19 @@ Graphical user interface for R interpreter will start in a new window:
...
 ```
 
-## 2.2. Installing *seroincidence* package
+## 2.2. Installing *serocalculator* package
 
 Since this is a new installation of R, the **serocalculator** package must be installed before first 
 use. The package can be installed either through a menu option in the graphical user interface (GUI) 
 or with a command executed in the R console. Both options are presented:
 
 ### 2.2.1. Package installation from the graphical user interface
```
Comment (d-morrison) [inline vignettes/installation.Rmd]: This subsection is out of date; remove?

**Review by d-morrison — CHANGES_REQUESTED** [review]
Looks good, except for subsection about installing from downloaded binary files, which I don't think is currently an available option. Let's remove it?

File: vignettes/installation.Rmd (hunk 1 above)
Comment (kristinawlai) [reply vignettes/installation.Rmd]: Replaced with updated installation instructions (or so I think I did...)

**Comment by d-morrison** [comment]
@kristinawlai let me know if you want help with the check errors?

**Comment by kristinawlai** [comment]
@d-morrison Yes please! I'd appreciate it if you could take a look. Thanks!

**Comment by d-morrison** [comment]
> @d-morrison Yes please! I'd appreciate it if you could take a look. Thanks!

@kristinawlai I think https://github.com/kristinawlai/serocalculator/pull/2 will solve this. The key error message was:

> Quitting from lines 91-93 [unnamed-chunk-7] (installation.Rmd)
> Error: Error: processing vignette 'installation.Rmd' failed with diagnostics:
> trying to use CRAN without setting a mirror

**Review by d-morrison — APPROVED** [review]
Looks good!

**Pull request merged**